pcsx2/gui/ConsoleLogger.cpp

```diff
@@ -406,6 +408,7 @@ ConsoleLogFrame::ConsoleLogFrame( MainEmuFrame *parent, const wxString& title, A
 	menuLog.Append(wxID_SAVE,	_("&Save..."),		_("Save log contents to file"));
 	menuLog.Append(wxID_CLEAR,	_("C&lear"),		_("Clear the log window contents"));
 	menuLog.AppendSeparator();
+	menuLog.AppendCheckItem(MenuId_AutoDock, _("Autodock"), _("Dock log window to main PCSX2 window"))->Check(m_conf.AutoDock);
 	menuLog.AppendSubMenu( &menuAppear, _("Appearance") );
```

Comment:
I think dock is more better, or at least more consistent with the other help messages.